zerver/tests/test_event_queue.py

```diff
@@ -110,7 +111,7 @@ def test_enqueue_notifications(self) -> None:
         email_notice, mobile_notice = self.check_will_notify(
             user_profile.id, message_id, private_message=False,
             mentioned=False, wildcard_mention_notify=False,
-            stream_push_notify=True, stream_email_notify=False,
+            stream_push_notify=True, stream_email_notify=False, topic_email_notify=False,
```

Comment:
I think it'd be nice if this commit included `topic_push_notify`, just so that we only churn this file once (and then we can just have a line for `topic_*_notify`.  Though I think these flags should probably be called e.g. `topic_follow_email_notify`.

Reply:
Sure, will squash after the review of `topic_follow_push_notify` commit.